tests/integration/routes/test_session.py

```diff
@@ -63,18 +63,39 @@ def test_head_request_on_session_signed_out(self):
         self.assertStatusOK()
 
     @freeze_time(TIME_TO_FREEZE)
-    def test_session_expiry(self):
+    def test_get_session_expiry_doesnt_extend_session(self):
         self.launchSurvey()
-        self.get("/session-expiry")
+        # Advance time by 20 mins...
+        with freeze_time(TIME_TO_FREEZE + timedelta(seconds=20 * 60)):
+            self.get("/session-expiry")
+            response = self.getResponseData()
+            parsed_json = json_loads(response)
+            # ... check that the session expiry time is not affected by
+            # the request, and is still 45mins from the start time
```

Comment:
It's fine, but naturally this might not be 45 minutes in the future as it can be set.

Reply:
True, I'll set it in the setup to fix it